incubator/java-microprofile/image/project/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN  /project/util/check_version build
 
 # Copy the user's application pom and resolve all dependencies
 COPY ./user-app/pom.xml /project/user-app/pom.xml
-RUN cd /project/user-app && mvn package -Dskip=true -DskipTests
+COPY ./validate.sh /project/user-app/validate.sh
+RUN cd /project/user-app && ./validate.sh  && mvn package -Dskip=true -DskipTests 
```

Comment:
I realize the `-Dskip=true` came from #485  not from your PR... but does that really do anything?   Was it supposed to be something else...`-DskipLibertyPackage` maybe?